docs/source/tutorials_source/sdk-integration-tutorial.py

```diff
@@ -172,10 +172,24 @@ def forward(self, x):
 # Use CMake (follow `these instructions <../runtime-build-and-cross-compilation.html#configure-the-cmake-build>`__ to set up cmake) to execute the Bundled Program to generate the ``ETDump``::
 #
 #       cd executorch
-#       rm -rf cmake-out && mkdir cmake-out && cd cmake-out && cmake -DEXECUTORCH_BUILD_SDK=1 -DEXECUTORCH_BUILD_EXTENSION_DATA_LOADER=1 ..
-#       cd ..
-#       cmake --build cmake-out -j8 -t sdk_example_runner
-#       ./cmake-out/examples/sdk/sdk_example_runner --bundled_program_path <bundled_program>
+#       rm -rf cmake-out
```

Comment:
The old instructions only needed a few lines, but this is too complicated to copy-and-paste; please add a script.

I put together a first pass in https://gist.github.com/dbort/55cb1aee2c5673aab4f61087c80363b4.

Though when I run it it fails during the second cmake with
```
CMake Error at /Users/dbort/local/executorch/build/Codegen.cmake:132 (add_library):
  add_library cannot create target "portable_ops_lib" because an imported
  target with the same name already exists.
Call Stack (most recent call first):
  CMakeLists.txt:57 (gen_operators_lib)
```

If you do keep this as pasteable instructions, you should remove the `local` keyword, since it's not useful in this context.

Reply:
Sounds good. I'll cherry-pick this PR first to unbreak the tutorial, then work on moving it to a script.